src/typings/react-feature-toggles/index.d.ts

```diff
@@ -0,0 +1,63 @@
+declare module '@paralleldrive/react-feature-toggles' {
+  function Feature(inactiveComponent: any, activeComponent: any, name: string, children?: any): JSX.Element
+  export { Feature }
+
+  function FeatureToggles({
+    features,
+  }?: {
+    readonly features?: ReadonlyArray<string>
+    readonly children?: any
+  }): JSX.Element
+  export { FeatureToggles }
+
+  interface Feature {
+    readonly name: string
+    readonly isActive: boolean
+    readonly dependencies?: ReadonlyArray<string>
+  }
+
+  function getActiveFeatures(features: ReadonlyArray<Feature>): ReadonlyArray<string>
+  export { getActiveFeatures }
+
+  interface Query {
+    readonly ft: string
+  }
+  function getQueryFeatures(query?: Query): ReadonlyArray<string>
+  export { getQueryFeatures }
+
+  function mergeFeatures(
+    currentFeatures: ReadonlyArray<string>,
+    restOfFeatures: ReadonlyArray<string>
```

Comment:
restOfFeatures should be:
`...restOfFeatures: ReadonlyArray<string>`
I am not sure how to do the rest operator.

Reply:
See https://github.com/Microsoft/TypeScript/issues/15972

Reply:
Ok cool! That seems to have worked! I noticed that the ReadonlyArray wasn't an option, just wasn't able to figure it out/find the answer after that. Thank you!

Reply:
Well, it worked on my local machine, but out lint rule changes it to ReadonlyArray<string>.

Reply:
I think it's possible to avoid this rule in this line if you need

Reply:
I updated to `...restOfFeatures: string[]` with the rule, but it is giving this error: ```Argument of type 'string[]' is not assignable to parameter of type 'string'.``` Trying to work that out before I merge.

Reply:
I think the correct solution was `...restOfFeatures: Array<ReadonlyArray<string>>`

Reply:
LGTM @lautarodragan do you agree?